src/web/components/MainMedia.tsx

```diff
@@ -55,93 +50,6 @@ const immersiveWrapper = css`
     overflow: hidden;
 `;
 
-function renderElement(
```

Comment:
This code is moved in `Element Renderer` with some additional props to support all use cases